README.md

```diff
@@ -1,14 +1,23 @@
 # <img src="./src/images/antimicrox.png" alt="Icon" width="60"/> AntiMicroX
-
-1. [Description](#description)  
-2. [License](#license)  
-3. [Installation](#installation)
-4. [Wiki](#wiki)
-5. [Command Line](#command-line) 
-6. [Testing Under Linux](#testing-under-linux)  
-7. [AntiMicroX Profiles](#antimicrox-profiles)
-8. [Support](#support)  
-
+- [Description](#description)
+- [License](#license)
+- [Installation](#installation)
+	-  [windows](#windows)
```

Comment:
You just added a glossary entry to paragraph, which takes literally 2 lines.

This does not make sense. That's why installation section does not have links to its subsections